pkg/get/get_test.go

```diff
@@ -6703,3 +6703,101 @@ func Test_DownloaSkupper(t *testing.T) {
 		}
 	}
 }
+
+func Test_DownloadKwok(t *testing.T) {
+	var (
+		tools       = MakeTools()
+		name        = "kwok"
+		toolVersion = "v0.4.0"
+		tool        = getTool(name, tools)
+	)
+
+	tests := []test{
+		{
+			os:      "linux",
+			version: toolVersion,
+			arch:    archARM64,
+			url:     `https://github.com/kubernetes-sigs/kwok/releases/download/v0.4.0/kwok-linux-arm64`,
+		},
+		{
+			os:      "linux",
+			version: toolVersion,
+			arch:    arch64bit,
+			url:     `https://github.com/kubernetes-sigs/kwok/releases/download/v0.4.0/kwok-linux-amd64`,
+		},
+		{
+			os:      "darwin",
+			version: toolVersion,
+			arch:    archDarwinARM64,
+			url:     `https://github.com/kubernetes-sigs/kwok/releases/download/v0.4.0/kwok-darwin-arm64`,
+		},
+		{
+			os:      "darwin",
+			version: toolVersion,
+			arch:    arch64bit,
+			url:     `https://github.com/kubernetes-sigs/kwok/releases/download/v0.4.0/kwok-darwin-amd64`,
+		},
+		{
+			os:      "ming",
```

Comment:
We don't tend to have Arm64 Windows apps as a rule.

But let's keep this in for now

Reply:
Since they already build them, I think it could be a good addition.